### ISSUE TITLE
[UnitTest] Setup 100% Coverage on all Repositoryes

### ISSUE THREAD

**Issue by Xazin** [CollActionteam/collaction_app#222]
TBD

**Comment by Xazin** [comment]
@yasht01 @sudo-saksham 
We can slowly start on this, feel free to just pick up anyone at any given time, and simply let eachother know when there's a new PR 👍

**Comment by sudo-saksham** [comment]
Yeah, cool. I prefer starting with the lower level. So maybe writing unit tests for model classes.

**Comment by Xazin** [comment]
> Yeah, cool. I prefer starting with the lower level. So maybe writing unit tests for model classes.

You can write Tests for the DTO's, more specifically from and to json.

I'll create another Issue with the DTOs we need to write tests for and what type of tests.

**Comment by Xazin** [comment]
@sudo-saksham #224
This one is fairly straightforward.
